<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="buttonLoadFile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>load file</string>
    </property>
   </widget>
   <widget class="QLabel" name="regionLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
color: black;
	font: 12pt &quot;Frank Ruhl Hofshi&quot;;
}</string>
    </property>
    <property name="text">
     <string>Region:</string>
    </property>
   </widget>
   <widget class="QLabel" name="columnLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
color: black;
	font: 12pt &quot;Frank Ruhl Hofshi&quot;;
}</string>
    </property>
    <property name="text">
     <string>Column:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="regionLineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
background-color: rgb(255, 255, 255);
}

	</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="columnLineEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>120</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
background-color: rgb(255, 255, 255);
}

	</string>
    </property>
   </widget>
   <widget class="QLabel" name="fileNameLine">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
color: black;
	font: 12pt &quot;Frank Ruhl Hofshi&quot;;
}</string>
    </property>
    <property name="text">
     <string>File Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="showFileName">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>541</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
color: black;
	font: 12pt &quot;Frank Ruhl Hofshi&quot;;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonLoadData">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>210</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>load data</string>
    </property>
   </widget>
   <widget class="QLabel" name="regionInformationLabel">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>70</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
color: black;
	font: 12pt &quot;Frank Ruhl Hofshi&quot;;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>70</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
color: black;
	font: 12pt &quot;Frank Ruhl Hofshi&quot;;
}</string>
    </property>
    <property name="text">
     <string>Region:</string>
    </property>
   </widget>
   <widget class="QLabel" name="fileErrorLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonCalculateMetrics">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>260</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>calculate metrics</string>
    </property>
   </widget>
   <widget class="QLabel" name="resultLabelShow">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>110</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>120</y>
      <width>751</width>
      <height>501</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QTableWidget 
{
 Background: #FFFFFF; 
 Border:none; 
color: rgb(95, 95, 95);
font: 11pt &quot;Frank Ruhl Hofshi&quot;;
}
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineMax">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>330</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
background-color: rgb(255, 255, 255);
}

	</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineMin">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>380</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
background-color: rgb(255, 255, 255);
}

	</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineMed">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>430</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
background-color: rgb(255, 255, 255);
}

	</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
color: black;
	font: 12pt &quot;Frank Ruhl Hofshi&quot;;
}</string>
    </property>
    <property name="text">
     <string>max:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
color: black;
	font: 12pt &quot;Frank Ruhl Hofshi&quot;;
}</string>
    </property>
    <property name="text">
     <string>min:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
color: black;
	font: 12pt &quot;Frank Ruhl Hofshi&quot;;
}</string>
    </property>
    <property name="text">
     <string>mediana:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
